docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -66,5 +66,7 @@ There you can find links to upgrade notes for other versions too.
               $debugNote = 'Order page should redirect by 302 as the cart is empty by default.';
               $config->changeDefaultRequestDataSet($debugNote)
         ```
-
+- make promo codes editable on separate page ([#1319](https://github.com/shopsys/shopsys/pull/1319))
+    - replace `PromoCodeInlineEditCest` with [`VatInlineEditCest`](https://github.com/shopsys/project-base/blob/v8.1.0/tests/ShopBundle/Acceptance/acceptance/VatInlineEditCest.php) in order to have inline edit still acceptance tested
```

Comment:
Should not link be changed during release of 8.1.0? Until then it should be linked to master right?

Reply:
I actually don't know. I can change it if you want.

Reply:
it would be nice